ui/ts/models/log.ts

```diff
@@ -46,11 +54,14 @@ module Models {
       });
 
       private _url(): string {
-        let url: string = "/_status/local/log";
-        if (this.level() != null) {
-          url += "/" + this.level();
+        let url: string = "/_status/logs/";
+        if (this.node() != null) {
+          url += this.node();
         }
         url += "?";
+        if (this.level() != null) {
+          url += "level=" + this.level() + "&";
```

Comment:
I think javascript has a parameter encoding function built in, I can't think of it off the top of my head.